x-pack/test/functional/apps/status_page/status_page.ts

```diff
@@ -18,6 +18,7 @@ export default function statusPageFunctonalTests({
     after(async () => await esArchiver.unload('empty_kibana'));
 
     it('allows user to navigate without authentication', async () => {
+      this.tags(['skipCloud']);
```

Comment:
as far as I know `tags` works only for `describe` context.

Reply:
@dmlemeshko great catch, yes the tag needs to be moved to describe.